package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@balancer-labs/frontend-v2",
-  "version": "1.53.8",
+  "version": "1.53.9",
```

Comment:
The latest release is actually already [1.53.9](https://github.com/balancer-labs/frontend-v2/releases/tag/1.53.9). Please update to 1.53.10.

Reply:
Updated

Reply:
We've just released 1.53.11 now, so needs another bump to 1.53.12